packages/lesswrong/components/events/modules/HighlightedEventCard.tsx

```diff
@@ -20,20 +18,37 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
     alignItems: 'center',
     maxWidth: 800,
     height: 350,
-    backgroundSize: 'cover',
     backgroundPosition: 'center',
     background: theme.palette.primary.main,
     textAlign: 'center',
     color: 'white',
     borderRadius: 0,
+    boxShadow: "0 1px 3px rgba(0, 0, 0, 0.1)",
     margin: 'auto',
     [theme.breakpoints.down('xs')]: {
       marginLeft: -4,
       marginRight: -4,
     }
   },
   content: {
-    overflow: 'visible'
+    position: 'relative',
+    background: 'inherit',
+    padding: '10px 20px',
+    overflow: 'visible',
+    '&::before': {
+      content: "''",
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      background: 'inherit',
+      filter: 'blur(12px)',
+    }
```

Comment:
Wow, this is a really impressive bit of css.

I have to recommend nevertheless that we blur the entire image, as I think it looks better. Lizka was of a mixed opinion.

You could also mix and match, with a light blur outside of the text area, and a heavier blur inside.

If you want to do it with cloudinary, you can add `e_blur:400` to the list of modifiers.

Reply:
Nope, I think blurring the whole image would be a tragedy. But in an effort to be fair, I will get some more opinions. :)

Reply:
@jpaddison3 I did the [science](https://cea-core.slack.com/archives/CB5D60VSB/p1643662246011139) and looks like you're right! :) I updated the img:

<img width="728" alt="Screen Shot 2022-02-01 at 9 49 35 PM" src="https://user-images.githubusercontent.com/9057804/152057295-1ea791b0-39be-443c-9edb-2bba35404806.png">